frontend/src/routes/Home/Overview/OverviewPage.test.tsx

```diff
@@ -28,7 +29,68 @@ it('should responsed with correct value for mapProviderFromLabel function', () =
     expect(mapProviderFromLabel('other')).toEqual('other')
 })
 
+const getAddonRequest = {
+    apiVersion: 'view.open-cluster-management.io/v1beta1',
+    kind: 'ManagedClusterView',
+    metadata: {
+        name: '46de65eb9b4a488e6744a0b264a076cc107fd55e',
+        namespace: 'local-cluster',
+        labels: {
+            viewName: '46de65eb9b4a488e6744a0b264a076cc107fd55e',
+        },
+    },
+    spec: {
+        scope: {
+            name: 'observability-controller',
+            resource: 'clustermanagementaddon.v1alpha1.addon.open-cluster-management.io',
+        },
+    },
+}
+
+const getAddonResponse = {
+    apiVersion: 'view.open-cluster-management.io/v1beta1',
+    kind: 'ManagedClusterView',
+    metadata: {
+        name: '46de65eb9b4a488e6744a0b264a076cc107fd55e',
+        namespace: 'local-cluster',
+        labels: {
+            viewName: '46de65eb9b4a488e6744a0b264a076cc107fd55e',
+        },
+    },
+    spec: {
+        scope: {
+            name: 'observability-controller',
+            resource: 'clustermanagementaddon.v1alpha1.addon.open-cluster-management.io',
+        },
+    },
+    status: {
+        conditions: [
+            {
+                message: 'Watching resources successfully',
+                reason: 'GetResourceProcessing',
+                status: 'True',
+                type: 'Processing',
+            },
+        ],
+    },
+}
+
+const deleteMCVRequest = {
+    apiVersion: 'view.open-cluster-management.io/v1beta1',
+    kind: 'ManagedClusterView',
+    metadata: {
+        name: '46de65eb9b4a488e6744a0b264a076cc107fd55e',
+        namespace: 'local-cluster',
+        labels: {
+            viewName: '46de65eb9b4a488e6744a0b264a076cc107fd55e',
+        },
+    },
+}
+
 it('should render overview page in loading state', async () => {
+    const deleteResourceNock = nockGet(getAddonRequest, getAddonResponse)
```

Comment:
Why is this called `deleteResourceNock` when it is mocking a GET?

Reply:
forgot to rename after pasting